lib/formBuilder.tsx

```diff
@@ -286,6 +286,10 @@ const _getElementInitialValue = (
       dynamicRow[subElementID] = _getElementInitialValue(subElement, language);
     });
     return [dynamicRow];
+  } else if (element.properties.fileType) {
+    // For file attachments, we need several values like the FileName, FileReader base64 object and File object
+    nestedObj = { file: null, src: null, name: "" };
```

Comment:
After reviewing the logic I think this could be simplified to:
```
const nestedObj: Record<string, unknown> = {};
....
} else if (element.properties.fileType {
  // For file attachements, we need several values like FileName, FileReader base64 object and File object
  return {file: null, src: null, name: ""}
} else ...
```
It just saves from having to recast the value of nestedObj

Reply:
Agreed! Thank you, I fixed it and will tweak this solution after introducing `multipart/form-data`